e2e_samples/parking_sensors_synapse/scripts/deploy_synapse_artifacts.sh

```diff
@@ -177,6 +177,14 @@ createTrigger () {
     az synapse trigger create --file @./synapse/workspace/trigger/"${name}".json --name="${name}" --workspace-name "${SYNAPSE_WORKSPACE_NAME}"
 }
 
+createSqlDatabase(){
+    echo "Creating the external_db sql database using Synapse Serverless SQL..."
+
+    # Create a external_db database using Synapse Serverless SQL
+    sqlcmd -U ${synapseAnalyticsSQLAdmin} -P ${SYNAPSE_SQL_PASSWORD} -S tcp:${SYNAPSE_WORKSPACE_NAME}-ondemand.sql.azuresynapse.net -d master -I -Q "CREATE DATABASE external_db;"
```

Comment:
Add to top of file under REQUIRED ENV VARIABLES that this env variable synapseAnalyticsSQLAdmin (or rather SYNAPSE_ANALYTICS_SQL_ADMIN -- follow casing convention of external env variables) is required